src/kem/sike/Makefile.am

```diff
@@ -3,7 +3,7 @@ noinst_LTLIBRARIES = libkemsike.la
 
 libkemsike_la_SOURCES = kem_sike.c P503/P503.c P751/P751.c
 
-libkemsike_la_CFLAGS = $(AM_CFLAGS) -w
+libkemsike_la_CFLAGS = $(AM_CFLAGS) -w -g
```

Comment:
Do you really want debugging symbols here?

Reply:
Oups, thanks for catching that; leftovers from a debugging session